parquet-cli/src/test/java/org/apache/parquet/cli/BaseCommandTest.java

```diff
@@ -45,24 +46,32 @@ public void setUp() {
 
   @Test
   public void qualifiedPathOnLinuxTest() throws IOException {
+    Assume.assumeFalse
+             (System.getProperty("os.name").toLowerCase().startsWith("win"));
     Path path = this.command.qualifiedPath(LINUX_FILE_PATH);
     Assert.assertEquals("test.parquet", path.getName());
   }
 
   @Test
   public void qualifiedPathOnWinTest() throws IOException {
+    Assume.assumeFalse
```

Comment:
Is it working on a Mac? Why assumeFalse? I think, it would be cleaner to execute tests on an actual OS even if some are not covered (e.g. Mac).

Reply:
@gszadovszky Thanks for your response.

I forgot about Mac ...
I don't have it now. Could anyone check this on Mac?

> Why assumeFalse? 

The reason why it uses assumeFalse, `org.apache.hadoop.fs.Path.initialize` throws IllegalArgumentException depending on the condition. 

| Path Style |  Linux Style  |  Windows Style |
| ---- | ---- | ---- |
| on Linux |  OK  |  IllegalArgumentException |
| on Windows |  OK  |  OK  |

For testing on Linux OS, I changed the code to not execute the Windows Path Style tests. 

> I think, it would be cleaner to execute tests on an actual OS even if some are not covered (e.g. Mac).

I agree. Should I eliminate test code that depends on OS?

Reply:
I don't have a Mac either but I think it should work similarly as on Linux.

There is no problem with OS dependent tests like this one where the behavior itself depends on the OS. What I didn't like is `assumeFalse` instead of `assumeTrue`. I mean that we should assume on an actual OS and not that we are not using one of them. It may lead to some unexpected situations. I think, it is better to control the execution on the OS we know how it should work.
So, I suggest executing Win related tests with assuming it is a Win OS and executing the others assuming Linux or Mac.

Reply:
@gszadovszky Thanks for your comment. I fix the test code. Could you review it again?